### ISSUE TITLE
Some confusion around the numeric column type

### ISSUE THREAD

**Issue by christopher-caldwell** [danvk/pg-to-ts#37]
I have been putting together a few different automation tools, and have found a discrepancy  with the `numeric` column type.

[This tool returns it](https://github.com/danvk/pg-to-ts/blob/d3e9b4cc05f04d0f6eb931e53d05d90311afb5e9/src/schemaPostgres.ts#L38) as a TS `number`, but [the node pg lib](https://github.com/brianc/node-postgres/issues/811#issuecomment-116644535) returns as `string`. 

This came up when I was using [PGTyped](https://github.com/adelsz/pgtyped/issues/287) and there was a generation mismatch. 

I'm not if the PG driver is the ultimate decision maker here, but it's definitely returning `numeric` columns as strings.

So in summary, do you agree, and if not, is there a way to provide a custom override for a given type to be generate whatever the user wants?

**Comment by josejooj** [comment]
This happens because a `DECIMAL` field can be greater than `Number.MAX_SAFE_INTEGER`, or loose precision when converting to JavaScript `IEEE 754/Number`, so node-postgres converts it to a string, it's just a dependency behavior conflict.

You can self-build your custom `pg-to-ts` codebase to output decimal as string, or convert decimal into a number using node-postgres:
```js
import { Client, types } from 'pg';

types.setTypeParser(types.builtins.NUMERIC, value => parseFloat(value));

export const SQLClient = new Client({
    connectionString: process.env.PGSQL_CONNECTION_URI,
    types
});
```
Use this type parser only if you have decimals less than `9007199254740991`, instead parse to BigInt.
Remember to handle float precision loss.
